### DISCUSSION TITLE
Platform specific image resource

### DISCUSSION THREAD

**Discussion by IXChinmayD** [dotnet/maui#21095] [Upgrading from Xamarin]
I am migrating from Xamarin Forms to .NET8 MAUI. There are different resource folders based on the platform for Xamarin Forms. But, we have different images with identical names and types for different platforms. How to achieve the same in .NET MAUI.

**Comment by davidortinau** [comment]
Put them in the Platforms folders mirroring where you locate them in Xamarin.Forms, or if you want to use Resources/Images you can use conditional syntax by TFM. See the Condition example below.

```xml
<ItemGroup>
	<!-- App Icon -->
	<MauiIcon Include="Resources\AppIcon\appicon.svg" ForegroundFile="Resources\AppIcon\appiconfg.svg" Color="#512BD4" />

	<!-- Splash Screen -->
	<MauiSplashScreen Include="Resources\Splash\splash.svg" Color="#512BD4" BaseSize="128,128" />

	<!-- Images -->
	<MauiImage Include="Resources\Images\*" />
	<MauiImage Update="Resources\Images\dotnet_bot.png" Resize="True" BaseSize="300,185" />

	<MauiImage Include="Resources\Images\iOS\**" Condition="$(TargetFramework.Contains('-ios'))" />
	<MauiImage Include="Resources\Images\Android\**" Condition="$(TargetFramework.Contains('-android'))" />

	<!-- Custom Fonts -->
	<MauiFont Include="Resources\Fonts\*" />

	<!-- Raw Assets (also remove the "Resources\Raw" prefix) -->
	<MauiAsset Include="Resources\Raw\**" LogicalName="%(RecursiveDir)%(Filename)%(Extension)" />
</ItemGroup>
```
<img width="227" alt="Screenshot 2024-03-19 at 9 42 50 AM" src="https://github.com/dotnet/maui/assets/41873/24599697-61cb-47b3-8c79-4584199eeff5">

## Documentation

https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/configure-multi-targeting?view=net-maui-8.0#configure-folder-based-multi-targeting

https://learn.microsoft.com/en-us/dotnet/maui/user-interface/images/images?view=net-maui-8.0

## Screenshots


![Screenshot_1710859275](https://github.com/dotnet/maui/assets/41873/26dd00a2-348d-470d-9785-d3e2c6a531f3)
![Simulator Screenshot - iPhone 15 Pro - 2024-03-19 at 09 39 53](https://github.com/dotnet/maui/assets/41873/b7287073-a020-478d-823f-8ce64fc291fc)

**Reply by IXChinmayD** [reply]
Thank you for your response
